android/build.gradle

```diff
@@ -26,5 +35,6 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    //noinspection GradleDynamicVersion
+    implementation "com.facebook.react:react-native:${_reactNativeVersion}"
```

Comment:
Recommend changing `implementation` to `compile`.

Reply:
To support older version of react native - older android sdk configuration?

Reply:
Actually, using compile is required to support React Native 0.56.  If you create a new react native project (0.56) and then install and link react-native-config from this fork, it will fail with the error displayed above.  It is possible if you have updated your gradle version, then it will not fail, but if you use the default gradle version that ships with React Native, it will fail.